airflow/www/templates/airflow/trigger.html

```diff
@@ -129,7 +129,19 @@ <h2>Trigger DAG: {{ dag_id }}</h2>
         <input type="text" class="form-control" placeholder="Run ID" name="run_id">
       </div>
     </div>
-    {% if recent_confs|length > 0 %}
+    {% if form_fields and recent_params|length > 0 %}
+    <div class="form-group row">
```

Comment:
Mhm, you really want to add two JSON elements here, to be able to trigger using a conf and a params dict? I'd propose to switch to params from conf in this PR, having two fields rather confuses which to use (in my eyes)